test/onnx/onnx_test_common.py

```diff
@@ -461,6 +461,9 @@ def _compare_pytorch_onnx_with_ort(
 # The max onnx opset version to test for
 MAX_ONNX_OPSET_VERSION = _constants.ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET
 TESTED_OPSETS = range(MIN_ONNX_OPSET_VERSION, MAX_ONNX_OPSET_VERSION + 1)
+ONNXRT_SUPPORTED_OPSETS = range(
```

Comment:
I should have changed this in https://github.com/pytorch/pytorch/pull/107829. May need to rebase?

Reply:
NVM